WeatherWebService/WeatherWebService/WeatherDetailsVIewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  WeatherDetailsVIewController.swift
+//  WeatherWebService
+//
+//  Created by komal lunkad on 29/09/16.
+//  Copyright © 2016 komal lunkad. All rights reserved.
+//
+
+import UIKit
+
+class WeatherDetailsViewController: UIViewController {
+    
+    @IBOutlet var labelLatitude: UILabel!
+    @IBOutlet var labelLongitude: UILabel!
+    @IBOutlet var labelMaxTemperature: UILabel!
+    @IBOutlet var labelMinTemperature: UILabel!
+    var weather:Weather? = Weather()
+    var cityName: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        callWeatherApi(cityName!)
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    //MARK:CallWeatherApi
+    /** 
+     Calls Api and fetches data
+     @param cityname: The input city whose data is to be fetched
+    */
+    func callWeatherApi(cityName:String) {
+        let url = Constants.ApiDetails.url
+        let appID = Constants.ApiDetails.appID
+        let urlWithParameters = url + "?q=\(cityName)&appid=\(appID)"
```

Comment:
add constants for "?q=\" and "&appid=\", consult ayush for this.Maybe something like, API_PARAM_APPID will be suitable